src/Command/Initializer/Profiles/blank/Init.php

```diff
@@ -0,0 +1,542 @@
+ * @author Temuri Takalandze <me@abgeo.dev>
+ */
+class Init extends AbstractInitProfile
+{
+    /**
+     * @var bool
+     */
+    private $useDefaultSetup;
+
+    /**
+     * @var string
+     */
+    private $templatesDirectory;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function __construct(InputInterface $input, OutputInterface $output, ?Config $config = null)
+    {
+        parent::__construct($input, $output, $config);
+
+        $this->templatesDirectory = __DIR__ . DIRECTORY_SEPARATOR . 'templates';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function run(string $projectDirectory, bool $useDefaultSetup = false): int
+    {
+        $this->useDefaultSetup = $useDefaultSetup;
+        $exitCode = null;
+
+        if (0 !== ($exitCode = parent::run($projectDirectory, $useDefaultSetup))) {
+            return $exitCode;
+        }
+
+        if (0 !== ($exitCode = $this->createStructure($projectDirectory))) {
+            return $exitCode;
+        }
+
+        if (0 === ($exitCode = $this->setupDependencies($projectDirectory))) {
+            $this->io->success('New Slim project successfully created. Please run `composer install`.');
+        }
+
+        return $exitCode;
+    }
+
+    /**
+     * Create basic directory and files structure.
+     *
+     * @param string $projectDirectory
+     *
+     * @return int The Exit Code.
+     */
+    protected function createStructure(string $projectDirectory): int
+    {
+        $phpunitTemplate = null;
+        $directoryFullPath = getcwd() . DIRECTORY_SEPARATOR . $projectDirectory;
+        $composerJsonContent = $this->readComposerJson($directoryFullPath);
+        $directoriesToCreate = [
+            'bootstrap' => $this->config ? $this->config->getBootstrapDir() : 'app',
+            'index'     => $this->config ? $this->config->getIndexDir() : 'public',
+            'source'    => $this->config ? $this->config->getSourceDir() : 'src',
+            'logs'      => 'logs',
+            'tests'     => 'tests',
+        ];
+        $filesToCreate = [
+            $directoriesToCreate['bootstrap'] . DIRECTORY_SEPARATOR . 'dependencies.php',
+            $directoriesToCreate['bootstrap'] . DIRECTORY_SEPARATOR . 'routes.php',
+            $directoriesToCreate['bootstrap'] . DIRECTORY_SEPARATOR . 'settings.php',
+
+            $directoriesToCreate['index'] . DIRECTORY_SEPARATOR .
+                ($this->config ? $this->config->getIndexFile() : 'index.php'),
+        ];
+
+        foreach ($directoriesToCreate as $directory) {
+            if (!is_dir($directoryFullPath . DIRECTORY_SEPARATOR . $directory)) {
+                mkdir($directoryFullPath . DIRECTORY_SEPARATOR . $directory, 0755, true);
+            }
+        }
+
+        foreach ($filesToCreate as $file) {
+            if (!is_file($directoryFullPath . DIRECTORY_SEPARATOR . $file)) {
+                touch($directoryFullPath . DIRECTORY_SEPARATOR . $file);
```

Comment:
We should check if this did not fail, shouldn't we?

If so, there are many other functions like `copy` that we should check.